js/node/package.json

```diff
@@ -1,41 +1,41 @@
 {
   "dependencies": {
-    "accepts": "1.3.5",
+    "accepts": "1.3.7",
+    "ansi_up": "5.0.1",
     "ansi-html": "0.0.7",
-    "ansi_up": "4.0.3",
     "aqb": "2.1.0",
     "babel-code-frame": "6.26.0",
     "chai": "3.5.0",
-    "chalk": "1.1.3",
+    "chalk": "4.1.1",
     "content-disposition": "0.5.3",
     "dedent": "0.7.0",
-    "error-stack-parser": "2.0.2",
+    "error-stack-parser": "2.0.6",
     "eslint": "5.16.0",
     "expect.js": "0.3.1",
     "extendible": "0.1.1",
     "graphql-sync": "0.6.2-sync",
-    "highlight.js": "9.15.6",
-    "http-errors": "1.7.2",
+    "highlight.js": "10.7.2",
+    "http-errors": "1.8.0",
     "i": "0.3.6",
     "iconv-lite": "0.4.24",
     "joi": "14.3.1",
     "joi-to-json-schema": "4.0.1",
-    "js-yaml": "3.13.1",
-    "lodash": "4.17.13",
+    "js-yaml": "4.1.0",
+    "lodash": "4.17.21",
     "marked": "0.6.2",
     "media-typer": "0.3.0",
-    "mime-types": "2.1.22",
+    "mime-types": "2.1.30",
     "minimatch": "3.0.4",
     "mocha": "6.1.3",
-    "netmask": "1.0.6",
-    "qs": "6.7.0",
-    "range-parser": "1.2.0",
+    "netmask": "2.0.2",
+    "qs": "6.10.1",
+    "range-parser": "1.2.1",
     "semver": "6.0.0",
     "sinon": "1.17.6",
     "statuses": "1.5.0",
-    "timezone": "1.0.22",
-    "type-is": "1.6.16",
-    "underscore": "1.9.1",
+    "timezone": "1.0.23",
+    "type-is": "1.6.18",
+    "underscore": "1.13.1",
```

Comment:
do we need underscore + lodash?

Reply:
Lodash is listed in our documentation. Underscore not.

Reply:
@hkernbach I could probably remove underscore entirely. Do we need to escalate this question or can I just throw the module out as part of the upgrade?

Reply:
IMHO having one library (lodash or underscore) is enough. Also I would say here: As lodash is officially listed, we're safe to remove underscore.

Reply:
I've removed underscore and marked.